openslides/motions/config_variables.py

```diff
@@ -39,6 +39,15 @@ def get_config_variables():
         subgroup='General')
 
     yield ConfigVariable(
+        name='motions_recommendations_by',
+        default_value='Recommendation committee',
+        label='Name of recommendation committee',
+        weight=312,
+        group='Motions',
+        subgroup='General',
+        translatable=True)
```

Comment:
Yes, better. 
Or maybe: Disable recommendation field if not any recommendation_label is set in signals.py?? But by default we will deliver the signals with recommendation_labels... And a "normal" admin does not know about changing signals.py... So disabling with empty string is easier  - but should explained in a help text. As well as the hint to change the lables in workflow in signals.py.